package.json

```
@@ -12,7 +12,11 @@
       "import": "./dist/lexer.js",
       "require": "./dist/lexer.cjs"
     },
-    "./js": "./dist/lexer.asm.js"
+    "./js": {
+      "types": "./types/lexer.d.ts",
+      "module": "./dist/lexer.asm.js",
+      "import": "./dist/lexer.asm.js"
```

This can just be `"default"` since we don't have any other option:

```suggestion
      "default": "./dist/lexer.asm.js"
```